src/codeu/chat/server/LogQueue.java

```diff
@@ -0,0 +1,52 @@
+package codeu.chat.server;
+
+
+
+import com.sun.scenario.effect.impl.sw.sse.SSEBlend_SRC_OUTPeer;
```

Comment:
What's this line about? I don't see it being used, and right now it doesn't build for me.

I'll try removing it in a branch of this branch that I'm using to test the medium features.